platform/viewer/src/App.js

```diff
@@ -175,8 +165,9 @@ class App extends Component {
 function _initExtensions(extensions, hotkeys) {
   const defaultExtensions = [
     GenericViewerCommands,
-    MeasurementsPanel,
     OHIFCornerstoneExtension,
+    // WARNING: MUST BE REGISTERED _AFTER_ OHIFCORNERSTONEEXTENSION
```

Comment:
Food for thought, we need to worry about dependency management at some point in the future, as it'll only get worse. We'll get so many GitHub issues of X doesn't work with Y:  -> "...swap the order". If each extention listed their dependencies we could sort the order and the user won't have to care too much when pulling down a new shiny from the web. We could also throw a warning if a dependency isn't provided? I digress.